src/rohan/StringScan.java

```diff
@@ -32,11 +32,10 @@ String getMaxLength(String[] arr) {
 	
 	void displayEvenChar(String str) {
 		System.out.println("Display Characters at odd index: ");
-		for(int i=0;i<str.length();i++) {
+		for(int i=1;i<str.length();i++) {
 			if(!(i%2==0))
```

Comment:
look at the requirement. we want to print even character , index starts from 0. Odd index represents even characters of the string. look at the expected output. 

input : Technocredits
output : ehordt
You should write if(index%2 != 0)